### PR TITLE
[full-ci] [tests-only]investigate webUISharingInternalUsers/shareWithUsers.feature:78

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
Test webUISharingInternalUsers/shareWithUsers.feature:78 was passing on retry. This PR fixes that.

## Related Issue

- Fixes https://github.com/owncloud/web/issues/5455

## Motivation and Context
Test webUISharingInternalUsers/shareWithUsers.feature:78 was failing in the first run in this line `And the user opens folder "simple-folder" using the webUI` but passing on retry. The test should've passed in the first run as it passes locally everytime. This PR addresses that issue and fixes it.

## How Has This Been Tested?

- locally
- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>webUISharingInternalUsers</strong> https://drone.owncloud.com/owncloud/web/17343/6/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>webUISharingInternalUsers</strong> https://drone.owncloud.com/owncloud/web/17344/6/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>webUISharingInternalUsers</strong> https://drone.owncloud.com/owncloud/web/17346/6/1
The following scenarios passed on retry:
- webUISharingInternalUsers/shareWithUsers.feature:848

**Comment by ownclouders** [comment]
Results for <strong>webUISharingInternalUsers</strong> https://drone.owncloud.com/owncloud/web/17372/10/1
The following scenarios passed on retry:
- webUISharingInternalUsers/shareWithUsers.feature:455

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/17376/23/1
The following scenarios passed on retry:
- webUISharingInternalGroupsToRootSharingIndicator/shareWithGroups.feature:104

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternalUsers1</strong> https://drone.owncloud.com/owncloud/web/17376/54/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternal</strong> https://drone.owncloud.com/owncloud/web/17378/38/1
The following scenarios passed on retry:
- webUISharingExternal/federationSharing.feature:364

File: tests/acceptance/pageObjects/FilesPageElement/filesList.js [hunk 1]
```diff
@@ -182,6 +182,7 @@ module.exports = {
 
       await this.useXpath()
         .click(this.getFileLinkSelectorByFileName(folder, 'folder'))
+        .waitForAjaxCallsToStartAndFinish()
         .useCss()
```
Comment (saw-jan) [inline tests/acceptance/pageObjects/FilesPageElement/filesList.js:186]: Does it need to wait after the folder has been clicked/opened?

Comment (SwikritiT) [reply tests/acceptance/pageObjects/FilesPageElement/filesList.js:186]: No, that's not needed. Reloading the page before clicking the folder fixes the issue. That must me leftover. I'll remove it

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by phil-davis — APPROVED** [review]

**Pull request merged**